standard/requirements/message-payload-cloudevents/REQ_type.adoc

```diff
@@ -0,0 +1,102 @@
+[[req_message-payload-cloudevents_type]]
+[requirement]
+====
+[%metadata]
+identifier:: /req/message-payload-cloudevents/type
+part:: CloudEvents SHALL provide the following ``type``, ``datacontenttype`` and ``dataschema`` ``data`` matrix:
+
+|===
+|``type``|``datacontenttype``|``dataschema``|``data``
+
+|`org.ogc.api.collection.create`
+|`application/json`
+|`https://raw.githubusercontent.com/opengeospatial/ogcapi-common/refs/heads/master/collections/openapi/schemas/common-geodata/collectionDesc.yaml`
+|JSON payload of a created collection description
+
+|`org.ogc.api.collection.replace`
+|`application/json`
+|`https://raw.githubusercontent.com/opengeospatial/ogcapi-common/refs/heads/master/collections/openapi/schemas/common-geodata/collectionDesc.yaml`
+|JSON payload of an updated (full) collection description
+
+
+|`org.ogc.api.collection.update`
+|`application/json`
+|`https://raw.githubusercontent.com/opengeospatial/ogcapi-common/refs/heads/master/collections/openapi/schemas/common-geodata/collectionDesc.yaml`
+|JSON payload of an updated (partial) collection description
+
+|`org.ogc.api.collection.delete`
+|`text/plain`
+|None
+|Identifier of deleted collection
+
+|`org.ogc.api.collection.item.create`
+|`application/json`
+|`https://raw.githubusercontent.com/opengeospatial/ogcapi-common/refs/heads/master/collections/openapi/schemas/common-geodata/collectionDesc.yaml`
+|JSON payload of a created collection item
+
+|`org.ogc.api.collection.item.replace`
+|`application/json`
+|`https://raw.githubusercontent.com/opengeospatial/ogcapi-common/refs/heads/master/collections/openapi/schemas/common-geodata/collectionDesc.yaml`
+|JSON payload of an updated (full) collection item
+
+|`org.ogc.api.collection.item.update`
+|`application/json`
+|`https://raw.githubusercontent.com/opengeospatial/ogcapi-common/refs/heads/master/collections/openapi/schemas/common-geodata/collectionDesc.yaml`
+|JSON payload of an updated (partial) collection item
+
+|`org.ogc.api.collection.item.delete`
+|`text/plain`
+|None
+|Identifier of deleted collection item
+
+|`org.ogc.api.collection.tile.create`
+|`text/plain`
+|None
+|Identifier of the created tile in x/y/z
+
+|`org.ogc.api.collection.tile.replace`
+|`text/plain`
+|None
+|Identifier of the updated (full) tile in x/y/z
+
+|`org.ogc.api.collection.tile.update`
+|`text/plain`
+|None
+|Identifier of the updated (partial) tile in x/y/z
+
+|`org.ogc.api.collection.tile.update`
+|`text/plain`
+|None
+|Identifier of deleted tile in x/y/z
+
+|`org.ogc.api.process.deploy`
```

Comment:
Is there any reason for not having the following type defined `org.ogc.apiproces.deplace`?  

It would be the same as the line for `org.ogc.api.process.deploy` for the other columns, but it makes sense to distinguish between deployment and replacement of a process.

Reply:
@gfenoy good catch!  Update pushed to this PR.